team-manager/src/main/java/ua/knu/knudev/teammanager/domain/Recruitment.java

```diff
@@ -0,0 +1,44 @@
+package ua.knu.knudev.teammanager.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.*;
+import ua.knu.knudev.knudevcommon.constant.Expertise;
+import ua.knu.knudev.knudevcommon.constant.RecruitmentStatus;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(schema = "team_management", name = "recruitment")
+@Builder
+public class Recruitment {
+
+    @Id
+    @Column(updatable = false, nullable = false)
+    private Integer recruitmentNumber;
```

Comment:
So, what will be better - to completely split this domain into 2 domains: `ActiveRecruitment`(which will receieve the most requests), `ClosedRecruitments`, which both will be inherited from some parent class

So `ActiveRecruitmen`t will have id `Expertise expertise`. And `ClosedRecruitments` will have composite id, consisting of number and experitse

When you create new ActiveRecruitmen, you create SCHEDULED TASKS ON IT ON LIMIT SPECIFIED DATE. IMPLEMENT THIS feature

PLEASE CONTACT ME FOR DETAILS TOMORROW, I VERY WANT TO GO TO SLEEP ::)))

We will make this domain better